### ISSUE TITLE
Cannot Launch from .desktop launcher

### ISSUE THREAD

**Issue by SimonaTV** [shutter-project/shutter#405]
#### Brief summary of issue
On Ubuntu 20.04 LTS can't run Shutter by launcher but only by terminal and by the list of error It shows, seems it cannot locate Gtk3.pm
I have install and reinstall solving every dependecies but nothing it's changed.

#### Steps to reproduce the issue

1. 
2. 
3. 

#### Error output when lunching shutter from terminal

<details>
  <summary>Output</summary>

```
shutter
Subroutine Pango::Layout::set_text redefined at /usr/share/perl5/Gtk3.pm line 2260.
	require Gtk3.pm called at /usr/bin/shutter line 72
	Shutter::App::BEGIN() called at /usr/share/perl5/Gtk3.pm line 2260
	eval {...} called at /usr/share/perl5/Gtk3.pm line 2260
Subroutine Pango::Layout::set_markup redefined at /usr/share/perl5/Gtk3.pm line 2266.
	require Gtk3.pm called at /usr/bin/shutter line 72
	Shutter::App::BEGIN() called at /usr/share/perl5/Gtk3.pm line 2266
	eval {...} called at /usr/share/perl5/Gtk3.pm line 2266
GLib-GObject-CRITICAL **: g_boxed_type_register_static: assertion 'g_type_from_name (name) == 0' failed at /usr/lib/x86_64-linux-gnu/perl5/5.30/Glib/Object/Introspection.pm line 110.
 at /usr/share/perl5/Gtk3.pm line 489.
	Gtk3::import("Gtk3", "-init") called at /usr/bin/shutter line 72
	Shutter::App::BEGIN() called at /usr/bin/shutter line 72
	eval {...} called at /usr/bin/shutter line 72
GLib-CRITICAL **: g_once_init_leave: assertion 'result != 0' failed at /usr/lib/x86_64-linux-gnu/perl5/5.30/Glib/Object/Introspection.pm line 110.
 at /usr/share/perl5/Gtk3.pm line 489.
	Gtk3::import("Gtk3", "-init") called at /usr/bin/shutter line 72
	Shutter::App::BEGIN() called at /usr/bin/shutter line 72
	eval {...} called at /usr/bin/shutter line 72
GLib-GObject-CRITICAL **: g_boxed_type_register_static: assertion 'g_type_from_name (name) == 0' failed at /usr/lib/x86_64-linux-gnu/perl5/5.30/Glib/Object/Introspection.pm line 110.
 at /usr/share/perl5/Gtk3.pm line 489.
	Gtk3::import("Gtk3", "-init") called at /usr/bin/shutter line 72
	Shutter::App::BEGIN() called at /usr/bin/shutter line 72
	eval {...} called at /usr/bin/shutter line 72
GLib-CRITICAL **: g_once_init_leave: assertion 'result != 0' failed at /usr/lib/x86_64-linux-gnu/perl5/5.30/Glib/Object/Introspection.pm line 110.
 at /usr/share/perl5/Gtk3.pm line 489.
	Gtk3::import("Gtk3", "-init") called at /usr/bin/shutter line 72
	Shutter::App::BEGIN() called at /usr/bin/shutter line 72
	eval {...} called at /usr/bin/shutter line 72
GLib-GObject-CRITICAL **: g_boxed_type_register_static: assertion 'g_type_from_name (name) == 0' failed at /usr/lib/x86_64-linux-gnu/perl5/5.30/Glib/Object/Introspection.pm line 110.
 at /usr/share/perl5/Gtk3.pm line 489.
	Gtk3::import("Gtk3", "-init") called at /usr/bin/shutter line 72
	Shutter::App::BEGIN() called at /usr/bin/shutter line 72
	eval {...} called at /usr/bin/shutter line 72
GLib-CRITICAL **: g_once_init_leave: assertion 'result != 0' failed at /usr/lib/x86_64-linux-gnu/perl5/5.30/Glib/Object/Introspection.pm line 110.
 at /usr/share/perl5/Gtk3.pm line 489.
	Gtk3::import("Gtk3", "-init") called at /usr/bin/shutter line 72
	Shutter::App::BEGIN() called at /usr/bin/shutter line 72
	eval {...} called at /usr/bin/shutter line 72
GLib-LOG **: unsetenv() is not thread-safe and should not be used after threads are created at /usr/share/perl5/Gtk3.pm line 555.
	Gtk3::init() called at /usr/share/perl5/Gtk3.pm line 2314
	Gtk3::__ANON__() called at /usr/share/perl5/Gtk3.pm line 513
	Gtk3::import("Gtk3", "-init") called at /usr/bin/shutter line 72
	Shutter::App::BEGIN() called at /usr/bin/shutter line 72
	eval {...} called at /usr/bin/shutter line 72
Variable "$progname_active" will not stay shared at /usr/bin/shutter line 2770.
Variable "$progname" will not stay shared at /usr/bin/shutter line 2771.
Variable "$im_colors_active" will not stay shared at /usr/bin/shutter line 2779.
Variable "$combobox_im_colors" will not stay shared at /usr/bin/shutter line 2780.
Variable "$trans_check" will not stay shared at /usr/bin/shutter line 2790.
Variable "$css_provider_alpha" will not stay shared at /usr/bin/shutter line 2791.
Variable "$trans_custom" will not stay shared at /usr/bin/shutter line 2796.
Variable "$trans_custom_btn" will not stay shared at /usr/bin/shutter line 2797.
Variable "$trans_backg" will not stay shared at /usr/bin/shutter line 2803.
Variable "$window" will not stay shared at /usr/bin/shutter line 2806.
Variable "$cursor_active" will not stay shared at /usr/bin/shutter line 2811.
Variable "$cursor_status_active" will not stay shared at /usr/bin/shutter line 2811.
Variable "$delay_status" will not stay shared at /usr/bin/shutter line 2821.
Variable "$delay" will not stay shared at /usr/bin/shutter line 2821.
Variable "$delay_vlabel" will not stay shared at /usr/bin/shutter line 2822.
Variable "$d" will not stay shared at /usr/bin/shutter line 2822.
Variable "$delay_status_vlabel" will not stay shared at /usr/bin/shutter line 2828.
Variable "$menu_delay_vlabel" will not stay shared at /usr/bin/shutter line 2833.
Variable "$menu_delay" will not stay shared at /usr/bin/shutter line 2833.
Variable "$hide_time_vlabel" will not stay shared at /usr/bin/shutter line 2838.
Variable "$hide_time" will not stay shared at /usr/bin/shutter line 2838.
Variable "$thumbnail_active" will not stay shared at /usr/bin/shutter line 2843.
Variable "$thumbnail" will not stay shared at /usr/bin/shutter line 2844.
Variable "$combobox_type" will not stay shared at /usr/bin/shutter line 2859.
Variable "$scale" will not stay shared at /usr/bin/shutter line 2860.
Variable "$bordereffect_active" will not stay shared at /usr/bin/shutter line 2872.
Variable "$bordereffect" will not stay shared at /usr/bin/shutter line 2873.
Variable "$bordereffect_vlabel" will not stay shared at /usr/bin/shutter line 2881.
Variable "$scale_label" will not stay shared at /usr/bin/shutter line 2888.
Variable "$combobox_ns" will not stay shared at /usr/bin/shutter line 2912.
Variable "$window" will not stay shared at /usr/bin/shutter line 2929.
Variable "$hide_active" will not stay shared at /usr/bin/shutter line 2933.
Variable "$is_hidden" will not stay shared at /usr/bin/shutter line 2937.
Variable "$wnck_screen" will not stay shared at /usr/bin/shutter line 2962.
Variable "$d" will not stay shared at /usr/bin/shutter line 2964.
Variable "$notify_ptimeout_active" will not stay shared at /usr/bin/shutter line 2988.
Variable "$menu_delay" will not stay shared at /usr/bin/shutter line 2990.
Variable "$hide_time" will not stay shared at /usr/bin/shutter line 3055.
Variable "%session_screens" will not stay shared at /usr/bin/shutter line 3078.
Variable "$notebook" will not stay shared at /usr/bin/shutter line 3080.
Variable "$lp" will not stay shared at /usr/bin/shutter line 3098.
Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 3127.
Variable "$close_at_close_active" will not stay shared at /usr/bin/shutter line 3142.
Variable "$tray" will not stay shared at /usr/bin/shutter line 3143.
Variable "$window" will not stay shared at /usr/bin/shutter line 3145.
Variable "$is_hidden" will not stay shared at /usr/bin/shutter line 3146.
Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 3162.
Variable "$combobox_settings_profiles" will not stay shared at /usr/bin/shutter line 3177.
Variable "$sas" will not stay shared at /usr/bin/shutter line 3181.
Variable "$fs_active" will not stay shared at /usr/bin/shutter line 3181.
Variable "$fs_min_active" will not stay shared at /usr/bin/shutter line 3181.
Variable "$fs_nonot_active" will not stay shared at /usr/bin/shutter line 3181.
Variable "$app" will not stay shared at /usr/bin/shutter line 3183.
Variable "$window" will not stay shared at /usr/bin/shutter line 3215.
Variable "$tray_menu" will not stay shared at /usr/bin/shutter line 3224.
Variable "$tray_menu" will not stay shared at /usr/bin/shutter line 3242.
Variable "$is_hidden" will not stay shared at /usr/bin/shutter line 3262.
Variable "$d" will not stay shared at /usr/bin/shutter line 3274.
Variable "$sm" will not stay shared at /usr/bin/shutter line 3288.
Variable "$sm" will not stay shared at /usr/bin/shutter line 3313.
Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 3344.
Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 3378.
Variable "%session_screens" will not stay shared at /usr/bin/shutter line 3402.
Variable "$clipboard" will not stay shared at /usr/bin/shutter line 3439.
Variable "%session_screens" will not stay shared at /usr/bin/shutter line 3475.
Variable "$notebook" will not stay shared at /usr/bin/shutter line 3484.
Variable "$sm" will not stay shared at /usr/bin/shutter line 3494.
Variable "$st" will not stay shared at /usr/bin/shutter line 3504.
Variable "$tray_menu" will not stay shared at /usr/bin/shutter line 3517.
Variable "$notebook" will not stay shared at /usr/bin/shutter line 3705.
Variable "%session_screens" will not stay shared at /usr/bin/shutter line 3705.
Variable "$settings_dialog" will not stay shared at /usr/bin/shutter line 3713.
Variable "$current_profile_indx" will not stay shared at /usr/bin/shutter line 3729.
Variable "$combobox_settings_profiles" will not stay shared at /usr/bin/shutter line 3729.
Variable "$sas" will not stay shared at /usr/bin/shutter line 3743.
Variable "$fs_active" will not stay shared at /usr/bin/shutter line 3743.
Variable "$fs_min_active" will not stay shared at /usr/bin/shutter line 3743.
Variable "$fs_nonot_active" will not stay shared at /usr/bin/shutter line 3743.
Variable "$pagesetup" will not stay shared at /usr/bin/shutter line 3761.
Variable "$window" will not stay shared at /usr/bin/shutter line 3761.
Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 3783.
Variable "$current_profile_indx" will not stay shared at /usr/bin/shutter line 3825.
Variable "$sas" will not stay shared at /usr/bin/shutter line 3838.
Variable "$fs_active" will not stay shared at /usr/bin/shutter line 3838.
Variable "$fs_min_active" will not stay shared at /usr/bin/shutter line 3838.
Variable "$fs_nonot_active" will not stay shared at /usr/bin/shutter line 3838.
Variable "$current_profile_indx" will not stay shared at /usr/bin/shutter line 3856.
Variable "$d" will not stay shared at /usr/bin/shutter line 3863.
Variable "$sd" will not stay shared at /usr/bin/shutter line 3865.
Variable "$settings_xml" will not stay shared at /usr/bin/shutter line 3876.
Variable "$current_profile_indx" will not stay shared at /usr/bin/shutter line 3877.
Variable "$d" will not stay shared at /usr/bin/shutter line 3884.
Variable "$notebook" will not stay shared at /usr/bin/shutter line 3898.
Variable "$d" will not stay shared at /usr/bin/shutter line 3918.
Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 3924.
Variable "$d" will not stay shared at /usr/bin/shutter line 3938.
Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 3963.
Variable "%session_screens" will not stay shared at /usr/bin/shutter line 3996.
Variable "$notebook" will not stay shared at /usr/bin/shutter line 4026.
Variable "%session_screens" will not stay shared at /usr/bin/shutter line 4067.
Variable "$sd" will not stay shared at /usr/bin/shutter line 4114.
Variable "$d" will not stay shared at /usr/bin/shutter line 4114.
Variable "@signal_connections" will not stay shared at /usr/bin/shutter line 4130.
Variable "$app" will not stay shared at /usr/bin/shutter line 4131.
Variable "$tray" will not stay shared at /usr/bin/shutter line 4137.
Variable "$st" will not stay shared at /usr/bin/shutter line 4178.
Variable "$sm" will not stay shared at /usr/bin/shutter line 4178.
Variable "$wnck_screen" will not stay shared at /usr/bin/shutter line 4182.
Variable "$window" will not stay shared at /usr/bin/shutter line 4229.
Variable "$is_hidden" will not stay shared at /usr/bin/shutter line 4241.
Variable "$st" will not stay shared at /usr/bin/shutter line 4247.
Variable "%session_screens" will not stay shared at /usr/bin/shutter line 4281.
Variable "$css_provider_alpha" will not stay shared at /usr/bin/shutter line 4284.
Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 4371.
Variable "%settings" will not stay shared at /usr/bin/shutter line 4458.
Variable "$combobox_settings_profiles" will not stay shared at /usr/bin/shutter line 4460.
Variable "$sm" will not stay shared at /usr/bin/shutter line 4464.
Variable "$combobox_type" will not stay shared at /usr/bin/shutter line 4472.
Variable "$scale" will not stay shared at /usr/bin/shutter line 4473.
Variable "$filename" will not stay shared at /usr/bin/shutter line 4474.
Variable "$saveDir_button" will not stay shared at /usr/bin/shutter line 4475.
Variable "$save_auto_active" will not stay shared at /usr/bin/shutter line 4479.
Variable "$save_ask_active" will not stay shared at /usr/bin/shutter line 4480.
Variable "$save_no_active" will not stay shared at /usr/bin/shutter line 4481.
Variable "$image_autocopy_active" will not stay shared at /usr/bin/shutter line 4482.
Variable "$fname_autocopy_active" will not stay shared at /usr/bin/shutter line 4483.
Variable "$no_autocopy_active" will not stay shared at /usr/bin/shutter line 4484.
Variable "$cursor_active" will not stay shared at /usr/bin/shutter line 4485.
Variable "$delay" will not stay shared at /usr/bin/shutter line 4486.
Variable "$current_monitor_active" will not stay shared at /usr/bin/shutter line 4489.
Variable "$st" will not stay shared at /usr/bin/shutter line 4493.
Variable "$progname" will not stay shared at /usr/bin/shutter line 4506.
Variable "$progname_active" will not stay shared at /usr/bin/shutter line 4515.
Variable "$combobox_im_colors" will not stay shared at /usr/bin/shutter line 4516.
Variable "$im_colors_active" will not stay shared at /usr/bin/shutter line 4517.
Variable "$thumbnail" will not stay shared at /usr/bin/shutter line 4518.
Variable "$thumbnail_active" will not stay shared at /usr/bin/shutter line 4519.
Variable "$bordereffect" will not stay shared at /usr/bin/shutter line 4520.
Variable "$bordereffect_active" will not stay shared at /usr/bin/shutter line 4521.
Variable "$bordereffect_cbtn" will not stay shared at /usr/bin/shutter line 4522.
Variable "$zoom_active" will not stay shared at /usr/bin/shutter line 4526.
Variable "$as_help_active" will not stay shared at /usr/bin/shutter line 4527.
Variable "$asel_size3" will not stay shared at /usr/bin/shutter line 4528.
Variable "$asel_size4" will not stay shared at /usr/bin/shutter line 4529.
Variable "$asel_size1" will not stay shared at /usr/bin/shutter line 4530.
Variable "$asel_size2" will not stay shared at /usr/bin/shutter line 4531.
Variable "$border_active" will not stay shared at /usr/bin/shutter line 4532.
Variable "$winresize_active" will not stay shared at /usr/bin/shutter line 4533.
Variable "$winresize_w" will not stay shared at /usr/bin/shutter line 4534.
Variable "$winresize_h" will not stay shared at /usr/bin/shutter line 4535.
Variable "$autoshape_active" will not stay shared at /usr/bin/shutter line 4536.
Variable "$visible_windows_active" will not stay shared at /usr/bin/shutter line 4537.
Variable "$menu_delay" will not stay shared at /usr/bin/shutter line 4538.
Variable "$menu_waround_active" will not stay shared at /usr/bin/shutter line 4539.
Variable "$combobox_web_width" will not stay shared at /usr/bin/shutter line 4540.
Variable "$trans_check" will not stay shared at /usr/bin/shutter line 4543.
Variable "$trans_custom" will not stay shared at /usr/bin/shutter line 4544.
Variable "$trans_custom_btn" will not stay shared at /usr/bin/shutter line 4545.
Variable "$trans_backg" will not stay shared at /usr/bin/shutter line 4547.
Variable "$session_asc" will not stay shared at /usr/bin/shutter line 4549.
Variable "$session_asc_combo" will not stay shared at /usr/bin/shutter line 4550.
Variable "$session_desc" will not stay shared at /usr/bin/shutter line 4551.
Variable "$session_desc_combo" will not stay shared at /usr/bin/shutter line 4552.
Variable "$fs_active" will not stay shared at /usr/bin/shutter line 4555.
Variable "$fs_min_active" will not stay shared at /usr/bin/shutter line 4556.
Variable "$fs_nonot_active" will not stay shared at /usr/bin/shutter line 4557.
Variable "$hide_active" will not stay shared at /usr/bin/shutter line 4558.
Variable "$hide_time" will not stay shared at /usr/bin/shutter line 4559.
Variable "$present_after_active" will not stay shared at /usr/bin/shutter line 4560.
Variable "$close_at_close_active" will not stay shared at /usr/bin/shutter line 4561.
Variable "$notify_after_active" will not stay shared at /usr/bin/shutter line 4562.
Variable "$notify_timeout_active" will not stay shared at /usr/bin/shutter line 4563.
Variable "$notify_ptimeout_active" will not stay shared at /usr/bin/shutter line 4564.
Variable "$combobox_ns" will not stay shared at /usr/bin/shutter line 4565.
Variable "$ask_on_delete_active" will not stay shared at /usr/bin/shutter line 4566.
Variable "$delete_on_close_active" will not stay shared at /usr/bin/shutter line 4567.
Variable "$ask_on_fs_delete_active" will not stay shared at /usr/bin/shutter line 4568.
Variable "$ftp_remote_entry" will not stay shared at /usr/bin/shutter line 4571.
Variable "$ftp_mode_combo" will not stay shared at /usr/bin/shutter line 4572.
Variable "$ftp_username_entry" will not stay shared at /usr/bin/shutter line 4573.
Variable "$ftp_password_entry" will not stay shared at /usr/bin/shutter line 4574.
Variable "$ftp_wurl_entry" will not stay shared at /usr/bin/shutter line 4575.
Variable "%plugins" will not stay shared at /usr/bin/shutter line 4578.
Variable "$sd" will not stay shared at /usr/bin/shutter line 4603.
Variable "$d" will not stay shared at /usr/bin/shutter line 4603.
Variable "%session_screens" will not stay shared at /usr/bin/shutter line 4611.
Variable "%accounts" will not stay shared at /usr/bin/shutter line 4637.
Variable "$combobox_type" will not stay shared at /usr/bin/shutter line 4687.
Variable "$int_jpeg" will not stay shared at /usr/bin/shutter line 4687.
Variable "$int_png" will not stay shared at /usr/bin/shutter line 4689.
Variable "$scale" will not stay shared at /usr/bin/shutter line 4698.
Variable "$filename" will not stay shared at /usr/bin/shutter line 4700.
Variable "$saveDir_button" will not stay shared at /usr/bin/shutter line 4703.
Variable "$save_auto_active" will not stay shared at /usr/bin/shutter line 4705.
Variable "$save_ask_active" will not stay shared at /usr/bin/shutter line 4706.
Variable "$save_no_active" will not stay shared at /usr/bin/shutter line 4707.
Variable "$image_autocopy_active" will not stay shared at /usr/bin/shutter line 4709.
Variable "$fname_autocopy_active" will not stay shared at /usr/bin/shutter line 4710.
Variable "$no_autocopy_active" will not stay shared at /usr/bin/shutter line 4711.
Variable "$cursor_active" will not stay shared at /usr/bin/shutter line 4713.
Variable "$delay" will not stay shared at /usr/bin/shutter line 4714.
Variable "$current_monitor_active" will not stay shared at /usr/bin/shutter line 4723.
Variable "$st" will not stay shared at /usr/bin/shutter line 4726.
Variable "$progname" will not stay shared at /usr/bin/shutter line 4741.
Variable "$progname_active" will not stay shared at /usr/bin/shutter line 4744.
Variable "$im_colors_active" will not stay shared at /usr/bin/shutter line 4746.
Variable "$combobox_im_colors" will not stay shared at /usr/bin/shutter line 4747.
Variable "$thumbnail" will not stay shared at /usr/bin/shutter line 4749.
Variable "$thumbnail_active" will not stay shared at /usr/bin/shutter line 4750.
Variable "$bordereffect" will not stay shared at /usr/bin/shutter line 4752.
Variable "$bordereffect_active" will not stay shared at /usr/bin/shutter line 4753.
Variable "$bordereffect_cbtn" will not stay shared at /usr/bin/shutter line 4755.
Variable "$zoom_active" will not stay shared at /usr/bin/shutter line 4759.
Variable "$as_help_active" will not stay shared at /usr/bin/shutter line 4761.
Variable "$asel_size3" will not stay shared at /usr/bin/shutter line 4763.
Variable "$asel_size4" will not stay shared at /usr/bin/shutter line 4764.
Variable "$asel_size1" will not stay shared at /usr/bin/shutter line 4765.
Variable "$asel_size2" will not stay shared at /usr/bin/shutter line 4766.
Variable "$border_active" will not stay shared at /usr/bin/shutter line 4768.
Variable "$winresize_active" will not stay shared at /usr/bin/shutter line 4770.
Variable "$winresize_w" will not stay shared at /usr/bin/shutter line 4771.
Variable "$winresize_h" will not stay shared at /usr/bin/shutter line 4772.
Variable "$autoshape_active" will not stay shared at /usr/bin/shutter line 4774.
Variable "$visible_windows_active" will not stay shared at /usr/bin/shutter line 4775.
Variable "$menu_waround_active" will not stay shared at /usr/bin/shutter line 4776.
Variable "$menu_delay" will not stay shared at /usr/bin/shutter line 4777.
Variable "$combobox_web_width" will not stay shared at /usr/bin/shutter line 4778.
Variable "$trans_check" will not stay shared at /usr/bin/shutter line 4781.
Variable "$trans_custom" will not stay shared at /usr/bin/shutter line 4782.
Variable "$trans_custom_btn" will not stay shared at /usr/bin/shutter line 4784.
Variable "$trans_backg" will not stay shared at /usr/bin/shutter line 4786.
Variable "$session_asc" will not stay shared at /usr/bin/shutter line 4788.
Variable "$session_asc_combo" will not stay shared at /usr/bin/shutter line 4789.
Variable "$session_desc" will not stay shared at /usr/bin/shutter line 4790.
Variable "$session_desc_combo" will not stay shared at /usr/bin/shutter line 4791.
Variable "$fs_active" will not stay shared at /usr/bin/shutter line 4794.
Variable "$fs_min_active" will not stay shared at /usr/bin/shutter line 4795.
Variable "$fs_nonot_active" will not stay shared at /usr/bin/shutter line 4796.
Variable "$hide_active" will not stay shared at /usr/bin/shutter line 4797.
Variable "$hide_time" will not stay shared at /usr/bin/shutter line 4798.
Variable "$present_after_active" will not stay shared at /usr/bin/shutter line 4799.
Variable "$close_at_close_active" will not stay shared at /usr/bin/shutter line 4800.
Variable "$notify_after_active" will not stay shared at /usr/bin/shutter line 4801.
Variable "$notify_timeout_active" will not stay shared at /usr/bin/shutter line 4802.
Variable "$notify_ptimeout_active" will not stay shared at /usr/bin/shutter line 4803.
Variable "$combobox_ns" will not stay shared at /usr/bin/shutter line 4804.
Variable "$ask_on_delete_active" will not stay shared at /usr/bin/shutter line 4805.
Variable "$delete_on_close_active" will not stay shared at /usr/bin/shutter line 4806.
Variable "$ask_on_fs_delete_active" will not stay shared at /usr/bin/shutter line 4807.
Variable "$ftp_remote_entry" will not stay shared at /usr/bin/shutter line 4816.
Variable "$ftp_mode_combo" will not stay shared at /usr/bin/shutter line 4817.
Variable "$ftp_username_entry" will not stay shared at /usr/bin/shutter line 4818.
Variable "$ftp_password_entry" will not stay shared at /usr/bin/shutter line 4819.
Variable "$ftp_wurl_entry" will not stay shared at /usr/bin/shutter line 4820.
Variable "$accounts_tree" will not stay shared at /usr/bin/shutter line 4834.
Variable "$accounts_model" will not stay shared at /usr/bin/shutter line 4836.
Variable "%plugins" will not stay shared at /usr/bin/shutter line 4874.
Variable "$sd" will not stay shared at /usr/bin/shutter line 4890.
Variable "$d" will not stay shared at /usr/bin/shutter line 4890.
Variable "$d" will not stay shared at /usr/bin/shutter line 4919.
Variable "$sd" will not stay shared at /usr/bin/shutter line 4970.
Variable "$d" will not stay shared at /usr/bin/shutter line 4970.
Variable "%accounts" will not stay shared at /usr/bin/shutter line 4980.
Variable "$status" will not stay shared at /usr/bin/shutter line 5054.
Variable "$d" will not stay shared at /usr/bin/shutter line 5066.
Variable "%session_screens" will not stay shared at /usr/bin/shutter line 5083.
Variable "$notebook" will not stay shared at /usr/bin/shutter line 5088.
Variable "$status" will not stay shared at /usr/bin/shutter line 5142.
Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 5146.
Variable "$d" will not stay shared at /usr/bin/shutter line 5162.
Variable "$sd" will not stay shared at /usr/bin/shutter line 5177.
Variable "%session_screens" will not stay shared at /usr/bin/shutter line 5188.
Variable "$d" will not stay shared at /usr/bin/shutter line 5189.
Variable "%session_screens" will not stay shared at /usr/bin/shutter line 5199.
Variable "$ask_on_delete_active" will not stay shared at /usr/bin/shutter line 5218.
Variable "$sd" will not stay shared at /usr/bin/shutter line 5219.
Variable "$d" will not stay shared at /usr/bin/shutter line 5220.
Variable "%session_screens" will not stay shared at /usr/bin/shutter line 5220.
Variable "$notebook" will not stay shared at /usr/bin/shutter line 5235.
Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 5240.
Variable "$window" will not stay shared at /usr/bin/shutter line 5250.
Variable "$is_hidden" will not stay shared at /usr/bin/shutter line 5250.
Variable "$delete_on_close_active" will not stay shared at /usr/bin/shutter line 5348.
Variable "%session_screens" will not stay shared at /usr/bin/shutter line 5353.
Variable "$notebook" will not stay shared at /usr/bin/shutter line 5359.
Variable "$d" will not stay shared at /usr/bin/shutter line 5360.
Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 5364.
Variable "$window" will not stay shared at /usr/bin/shutter line 5374.
Variable "$is_hidden" will not stay shared at /usr/bin/shutter line 5374.
Variable "$clipboard" will not stay shared at /usr/bin/shutter line 5451.
Variable "$combobox_type" will not stay shared at /usr/bin/shutter line 5455.
Variable "$sd" will not stay shared at /usr/bin/shutter line 5458.
Variable "$d" will not stay shared at /usr/bin/shutter line 5458.
Variable "$sp" will not stay shared at /usr/bin/shutter line 5468.
Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 5497.
Variable "$lp" will not stay shared at /usr/bin/shutter line 5518.
Variable "%session_screens" will not stay shared at /usr/bin/shutter line 5518.
Variable "$clipboard" will not stay shared at /usr/bin/shutter line 5531.
Variable "$d" will not stay shared at /usr/bin/shutter line 5532.
Variable "%session_screens" will not stay shared at /usr/bin/shutter line 5556.
Variable "%session_screens" will not stay shared at /usr/bin/shutter line 5578.
Variable "$d" will not stay shared at /usr/bin/shutter line 5594.
Variable "$sd" will not stay shared at /usr/bin/shutter line 5601.
Variable "%session_screens" will not stay shared at /usr/bin/shutter line 5630.
Variable "$d" will not stay shared at /usr/bin/shutter line 5645.
Variable "$sd" will not stay shared at /usr/bin/shutter line 5652.
Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 5673.
Variable "%plugins" will not stay shared at /usr/bin/shutter line 5689.
Variable "$sd" will not stay shared at /usr/bin/shutter line 5690.
Variable "$d" will not stay shared at /usr/bin/shutter line 5690.
Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 5699.
Variable "%session_screens" will not stay shared at /usr/bin/shutter line 5726.
Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 5732.
Variable "%session_screens" will not stay shared at /usr/bin/shutter line 5766.
Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 5771.
Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 5802.
Variable "%session_screens" will not stay shared at /usr/bin/shutter line 5819.
Variable "$scale" will not stay shared at /usr/bin/shutter line 5836.
Variable "$filename" will not stay shared at /usr/bin/shutter line 5845.
Variable "$saveDir_button" will not stay shared at /usr/bin/shutter line 5851.
Variable "$save_no_active" will not stay shared at /usr/bin/shutter line 5852.
Variable "$combobox_type" will not stay shared at /usr/bin/shutter line 5863.
Variable "$sd" will not stay shared at /usr/bin/shutter line 5866.
Variable "$d" will not stay shared at /usr/bin/shutter line 5866.
Variable "$delay" will not stay shared at /usr/bin/shutter line 5881.
Variable "$cursor_active" will not stay shared at /usr/bin/shutter line 5897.
Variable "$notify_timeout_active" will not stay shared at /usr/bin/shutter line 5909.
Variable "$wnck_screen" will not stay shared at /usr/bin/shutter line 5910.
Variable "$current_monitor_active" will not stay shared at /usr/bin/shutter line 5911.
Variable "$border_active" will not stay shared at /usr/bin/shutter line 5936.
Variable "$winresize_active" will not stay shared at /usr/bin/shutter line 5936.
Variable "$winresize_w" will not stay shared at /usr/bin/shutter line 5936.
Variable "$winresize_h" will not stay shared at /usr/bin/shutter line 5936.
Variable "$hide_time" will not stay shared at /usr/bin/shutter line 5937.
Variable "$autoshape_active" will not stay shared at /usr/bin/shutter line 5937.
Variable "$is_hidden" will not stay shared at /usr/bin/shutter line 5947.
Variable "$visible_windows_active" will not stay shared at /usr/bin/shutter line 5948.
Variable "$menu_waround_active" will not stay shared at /usr/bin/shutter line 5948.
Variable "$zoom_active" will not stay shared at /usr/bin/shutter line 5975.
Variable "$as_help_active" will not stay shared at /usr/bin/shutter line 5975.
Variable "$asel_size3" will not stay shared at /usr/bin/shutter line 5975.
Variable "$asel_size4" will not stay shared at /usr/bin/shutter line 5976.
Variable "$asel_size1" will not stay shared at /usr/bin/shutter line 5976.
Variable "$asel_size2" will not stay shared at /usr/bin/shutter line 5976.
Variable "$combobox_web_width" will not stay shared at /usr/bin/shutter line 5987.
Variable "$st" will not stay shared at /usr/bin/shutter line 5995.
Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 6135.
Variable "%session_screens" will not stay shared at /usr/bin/shutter line 6148.
Variable "$present_after_active" will not stay shared at /usr/bin/shutter line 6223.
Variable "$bordereffect_active" will not stay shared at /usr/bin/shutter line 6328.
Variable "$bordereffect" will not stay shared at /usr/bin/shutter line 6334.
Variable "$bordereffect_cbtn" will not stay shared at /usr/bin/shutter line 6334.
Variable "$save_ask_active" will not stay shared at /usr/bin/shutter line 6338.
Variable "$sp" will not stay shared at /usr/bin/shutter line 6364.
Variable "$im_colors_active" will not stay shared at /usr/bin/shutter line 6376.
Variable "$combobox_im_colors" will not stay shared at /usr/bin/shutter line 6378.
Variable "$thumbnail_active" will not stay shared at /usr/bin/shutter line 6391.
Variable "$thumbnail" will not stay shared at /usr/bin/shutter line 6394.
Variable "$lp" will not stay shared at /usr/bin/shutter line 6398.
Variable "$no_autocopy_active" will not stay shared at /usr/bin/shutter line 6444.
Variable "$image_autocopy_active" will not stay shared at /usr/bin/shutter line 6447.
Variable "$clipboard" will not stay shared at /usr/bin/shutter line 6448.
Variable "$fname_autocopy_active" will not stay shared at /usr/bin/shutter line 6452.
Variable "$progname_active" will not stay shared at /usr/bin/shutter line 6459.
Variable "$progname" will not stay shared at /usr/bin/shutter line 6460.
Variable "$notify_after_active" will not stay shared at /usr/bin/shutter line 6474.
Variable "$tray_menu" will not stay shared at /usr/bin/shutter line 6529.
Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 6553.
Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 6585.
Variable "%session_screens" will not stay shared at /usr/bin/shutter line 6591.
Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 6615.
Variable "%session_screens" will not stay shared at /usr/bin/shutter line 6621.
Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 6642.
Variable "%session_screens" will not stay shared at /usr/bin/shutter line 6648.
Variable "$pagesetup" will not stay shared at /usr/bin/shutter line 6661.
Variable "$lp" will not stay shared at /usr/bin/shutter line 6687.
Variable "$window" will not stay shared at /usr/bin/shutter line 6704.
Variable "$d" will not stay shared at /usr/bin/shutter line 6729.
Variable "%session_screens" will not stay shared at /usr/bin/shutter line 6729.
Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 6745.
Variable "%session_screens" will not stay shared at /usr/bin/shutter line 6777.
Variable "$window" will not stay shared at /usr/bin/shutter line 6788.
Variable "$sd" will not stay shared at /usr/bin/shutter line 6807.
Variable "$d" will not stay shared at /usr/bin/shutter line 6807.
Variable "$status" will not stay shared at /usr/bin/shutter line 6891.
Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 6892.
Variable "%session_screens" will not stay shared at /usr/bin/shutter line 6931.
Variable "$window" will not stay shared at /usr/bin/shutter line 6939.
Variable "$status" will not stay shared at /usr/bin/shutter line 6944.
Variable "$d" will not stay shared at /usr/bin/shutter line 6945.
Variable "$hu" will not stay shared at /usr/bin/shutter line 6945.
Variable "$combobox_settings_profiles" will not stay shared at /usr/bin/shutter line 6961.
Variable "$tray" will not stay shared at /usr/bin/shutter line 6962.
Variable "%session_screens" will not stay shared at /usr/bin/shutter line 6982.
Variable "%session_screens" will not stay shared at /usr/bin/shutter line 7003.
Variable "$notebook" will not stay shared at /usr/bin/shutter line 7026.
Variable "%session_screens" will not stay shared at /usr/bin/shutter line 7032.
Variable "%session_screens" will not stay shared at /usr/bin/shutter line 7046.
Variable "$notebook" will not stay shared at /usr/bin/shutter line 7056.
Variable "%session_screens" will not stay shared at /usr/bin/shutter line 7062.
Variable "$tray_menu" will not stay shared at /usr/bin/shutter line 7081.
Variable "$sm" will not stay shared at /usr/bin/shutter line 7090.
Variable "$combobox_status_profiles" will not stay shared at /usr/bin/shutter line 7096.
Variable "$combobox_status_profiles_label" will not stay shared at /usr/bin/shutter line 7099.
Variable "$d" will not stay shared at /usr/bin/shutter line 7105.
Variable "$status" will not stay shared at /usr/bin/shutter line 7107.
Variable "%session_screens" will not stay shared at /usr/bin/shutter line 7167.
Variable "$lp_ne" will not stay shared at /usr/bin/shutter line 7226.
Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 7394.
Variable "$ask_on_fs_delete_active" will not stay shared at /usr/bin/shutter line 7423.
Variable "$sd" will not stay shared at /usr/bin/shutter line 7430.
Variable "$d" will not stay shared at /usr/bin/shutter line 7431.
Variable "$notebook" will not stay shared at /usr/bin/shutter line 7439.
Variable "$sp" will not stay shared at /usr/bin/shutter line 7485.
Variable "%session_screens" will not stay shared at /usr/bin/shutter line 7623.
Variable "$lp" will not stay shared at /usr/bin/shutter line 7639.
Variable "$progname" will not stay shared at /usr/bin/shutter line 7648.
Variable "$combobox_settings_profiles" will not stay shared at /usr/bin/shutter line 7698.
Variable "$sd" will not stay shared at /usr/bin/shutter line 7813.
Variable "$d" will not stay shared at /usr/bin/shutter line 7814.
Variable "$window" will not stay shared at /usr/bin/shutter line 7849.
Variable "$d" will not stay shared at /usr/bin/shutter line 7849.
Variable "$lp" will not stay shared at /usr/bin/shutter line 7877.
Variable "%plugins" will not stay shared at /usr/bin/shutter line 7895.
Variable "$window" will not stay shared at /usr/bin/shutter line 7996.
Variable "$d" will not stay shared at /usr/bin/shutter line 7996.
Variable "%accounts" will not stay shared at /usr/bin/shutter line 8036.
Variable "$progname" will not stay shared at /usr/bin/shutter line 8137.
Variable "$current_monitor_active" will not stay shared at /usr/bin/shutter line 8148.
Variable "$d" will not stay shared at /usr/bin/shutter line 8148.
Variable "$settings_xml" will not stay shared at /usr/bin/shutter line 8253.
Variable "$wnck_screen" will not stay shared at /usr/bin/shutter line 8286.
Variable "$lp" will not stay shared at /usr/bin/shutter line 8296.
Variable "$d" will not stay shared at /usr/bin/shutter line 8299.
Variable "$d" will not stay shared at /usr/bin/shutter line 8347.
Variable "$wnck_screen" will not stay shared at /usr/bin/shutter line 8348.
Variable "$lp" will not stay shared at /usr/bin/shutter line 8357.
Variable "$is_hidden" will not stay shared at /usr/bin/shutter line 8474.
Variable "%session_start_screen" will not stay shared at /usr/bin/shutter line 8555.
Variable "$sm" will not stay shared at /usr/bin/shutter line 8567.
Variable "%session_screens" will not stay shared at /usr/bin/shutter line 8573.
Variable "$d" will not stay shared at /usr/bin/shutter line 8653.
Variable "$settings_xml" will not stay shared at /usr/bin/shutter line 8710.
Variable "$current_profile_indx" will not stay shared at /usr/bin/shutter line 8751.
Variable "$accounts_model" will not stay shared at /usr/bin/shutter line 8769.
Variable "%accounts" will not stay shared at /usr/bin/shutter line 8774.
Variable "%plugins" will not stay shared at /usr/bin/shutter line 8794.
Variable "$lp" will not stay shared at /usr/bin/shutter line 8811.
Variable "$d" will not stay shared at /usr/bin/shutter line 8843.
Variable "%accounts" will not stay shared at /usr/bin/shutter line 8868.
Variable "$accounts_model" will not stay shared at /usr/bin/shutter line 8873.
Variable "$d" will not stay shared at /usr/bin/shutter line 8964.
Variable "$window" will not stay shared at /usr/bin/shutter line 9203.
Variable "$d" will not stay shared at /usr/bin/shutter line 9205.
Variable "%session_screens" will not stay shared at /usr/bin/shutter line 9209.
Variable "$sd" will not stay shared at /usr/bin/shutter line 9318.
Variable "$d" will not stay shared at /usr/bin/shutter line 9437.
Variable "$window" will not stay shared at /usr/bin/shutter line 9437.
Variable "$lp_ne" will not stay shared at /usr/bin/shutter line 9453.
Variable "%session_screens" will not stay shared at /usr/bin/shutter line 9495.
Variable "$saveDir_button" will not stay shared at /usr/bin/shutter line 9496.
Variable "$sm" will not stay shared at /usr/bin/shutter line 9520.
Variable "%session_screens" will not stay shared at /usr/bin/shutter line 9542.
Variable "$d" will not stay shared at /usr/bin/shutter line 9545.
Variable "$window" will not stay shared at /usr/bin/shutter line 9546.
Variable "$saveDir_button" will not stay shared at /usr/bin/shutter line 9562.
Variable "$lp_ne" will not stay shared at /usr/bin/shutter line 9578.
Variable "$sp" will not stay shared at /usr/bin/shutter line 9734.
Variable "$sd" will not stay shared at /usr/bin/shutter line 9783.
Variable "$d" will not stay shared at /usr/bin/shutter line 9876.
Variable "$window" will not stay shared at /usr/bin/shutter line 9876.
Variable "%plugins" will not stay shared at /usr/bin/shutter line 9896.
Variable "$lp" will not stay shared at /usr/bin/shutter line 9982.
Variable "$sd" will not stay shared at /usr/bin/shutter line 10021.
Variable "$d" will not stay shared at /usr/bin/shutter line 10071.
Variable "$window" will not stay shared at /usr/bin/shutter line 10072.
Variable "%accounts" will not stay shared at /usr/bin/shutter line 10082.
Variable "$ftp_remote_entry" will not stay shared at /usr/bin/shutter line 10183.
Variable "$ftp_mode_combo" will not stay shared at /usr/bin/shutter line 10197.
Variable "$ftp_username_entry" will not stay shared at /usr/bin/shutter line 10209.
Variable "$ftp_password_entry" will not stay shared at /usr/bin/shutter line 10223.
Variable "$ftp_wurl_entry" will not stay shared at /usr/bin/shutter line 10235.
Variable "%session_screens" will not stay shared at /usr/bin/shutter line 10361.
Variable "$clipboard" will not stay shared at /usr/bin/shutter line 10462.
Variable "$d" will not stay shared at /usr/bin/shutter line 10663.
Variable "$d" will not stay shared at /usr/bin/shutter line 10683.
Variable "$window" will not stay shared at /usr/bin/shutter line 10698.
Variable "$d" will not stay shared at /usr/bin/shutter line 10704.
Variable "$sd" will not stay shared at /usr/bin/shutter line 10765.
Variable "%session_screens" will not stay shared at /usr/bin/shutter line 10789.
Variable "%session_screens" will not stay shared at /usr/bin/shutter line 10796.
Variable "%session_screens" will not stay shared at /usr/bin/shutter line 10803.
Variable "%session_screens" will not stay shared at /usr/bin/shutter line 10810.
Variable "$window" will not stay shared at /usr/bin/shutter line 10818.
Variable "$nav_toolbar" will not stay shared at /usr/bin/shutter line 10828.
Name "Gtk3::Gdk::SELECTION_CLIPBOARD" used only once: possible typo at /usr/bin/shutter line 290.
Useless use of hash element in void context at /usr/share/perl5/Shutter/App/Common.pm line 77.
	require Shutter/App/Common.pm called at /usr/bin/shutter line 206
Useless use of hash element in void context at /usr/share/perl5/Shutter/App/Common.pm line 80.
	require Shutter/App/Common.pm called at /usr/bin/shutter line 206
Subroutine lookup redefined at /usr/share/perl5/Shutter/Draw/DrawingTool.pm line 28.
	require Shutter/Draw/DrawingTool.pm called at /usr/bin/shutter line 228
Variable "$self" will not stay shared at /usr/share/perl5/Shutter/Draw/DrawingTool.pm line 671.
	require Shutter/Draw/DrawingTool.pm called at /usr/bin/shutter line 228
Variable "$self" will not stay shared at /usr/share/perl5/Shutter/Screenshot/SelectorAdvanced.pm line 841.
	require Shutter/Screenshot/SelectorAdvanced.pm called at /usr/bin/shutter line 233
GLib-LOG **: unsetenv() is not thread-safe and should not be used after threads are created at /usr/lib/x86_64-linux-gnu/perl5/5.30/Glib/Object/Introspection.pm line 246.
 at /usr/bin/shutter line 268.
	Shutter::App::STARTUP(Shutter::App=HASH(0x55d7b7e1f350)) called at /usr/lib/x86_64-linux-gnu/perl5/5.30/Glib/Object/Introspection.pm line 67
	Glib::Object::Introspection::__ANON__(Shutter::App=HASH(0x55d7b7e1f350)) called at /usr/bin/shutter line 10849
GLib-CRITICAL **: g_variant_type_checked_: assertion 'g_variant_type_string_is_valid (type_string)' failed at /usr/lib/x86_64-linux-gnu/perl5/5.30/Glib/Object/Introspection.pm line 67, <DATA> line 19.
 at /usr/bin/shutter line 313, <DATA> line 19.
	Shutter::App::STARTUP(Shutter::App=HASH(0x55d7b7e1f350)) called at /usr/lib/x86_64-linux-gnu/perl5/5.30/Glib/Object/Introspection.pm line 67
	Glib::Object::Introspection::__ANON__(Shutter::App=HASH(0x55d7b7e1f350)) called at /usr/bin/shutter line 10849
GLib-CRITICAL **: g_variant_type_copy: assertion 'g_variant_type_check (type)' failed at /usr/lib/x86_64-linux-gnu/perl5/5.30/Glib/Object/Introspection.pm line 67, <DATA> line 19.
 at /usr/bin/shutter line 313, <DATA> line 19.
	Shutter::App::STARTUP(Shutter::App=HASH(0x55d7b7e1f350)) called at /usr/lib/x86_64-linux-gnu/perl5/5.30/Glib/Object/Introspection.pm line 67
	Glib::Object::Introspection::__ANON__(Shutter::App=HASH(0x55d7b7e1f350)) called at /usr/bin/shutter line 10849
GLib-CRITICAL **: g_variant_type_checked_: assertion 'g_variant_type_string_is_valid (type_string)' failed at /usr/lib/x86_64-linux-gnu/perl5/5.30/Glib/Object/Introspection.pm line 67, <DATA> line 19.
 at /usr/bin/shutter line 333, <DATA> line 19.
	Shutter::App::STARTUP(Shutter::App=HASH(0x55d7b7e1f350)) called at /usr/lib/x86_64-linux-gnu/perl5/5.30/Glib/Object/Introspection.pm line 67
	Glib::Object::Introspection::__ANON__(Shutter::App=HASH(0x55d7b7e1f350)) called at /usr/bin/shutter line 10849
GLib-CRITICAL **: g_variant_type_copy: assertion 'g_variant_type_check (type)' failed at /usr/lib/x86_64-linux-gnu/perl5/5.30/Glib/Object/Introspection.pm line 67, <DATA> line 19.
 at /usr/bin/shutter line 333, <DATA> line 19.
	Shutter::App::STARTUP(Shutter::App=HASH(0x55d7b7e1f350)) called at /usr/lib/x86_64-linux-gnu/perl5/5.30/Glib/Object/Introspection.pm line 67
	Glib::Object::Introspection::__ANON__(Shutter::App=HASH(0x55d7b7e1f350)) called at /usr/bin/shutter line 10849
GLib-CRITICAL **: g_variant_type_checked_: assertion 'g_variant_type_string_is_valid (type_string)' failed at /usr/lib/x86_64-linux-gnu/perl5/5.30/Glib/Object/Introspection.pm line 67, <DATA> line 19.
 at /usr/bin/shutter line 340, <DATA> line 19.
	Shutter::App::STARTUP(Shutter::App=HASH(0x55d7b7e1f350)) called at /usr/lib/x86_64-linux-gnu/perl5/5.30/Glib/Object/Introspection.pm line 67
	Glib::Object::Introspection::__ANON__(Shutter::App=HASH(0x55d7b7e1f350)) called at /usr/bin/shutter line 10849
GLib-CRITICAL **: g_variant_type_copy: assertion 'g_variant_type_check (type)' failed at /usr/lib/x86_64-linux-gnu/perl5/5.30/Glib/Object/Introspection.pm line 67, <DATA> line 19.
```

</details>

#### Extra information, such as Shutter version, operating system and ideas for how to solve:
Shutter latest version 0.99
Distributor ID:	Ubuntu
Description:	Ubuntu 20.04.3 LTS
Release:	20.04
Codename:	focal

**Comment by yanhuacuo** [comment]
ME TOO

**Comment by Photon89** [comment]
Since Shutter runs fine from the terminal, the output gives no hints on the source of the problem. It shows many messages but none of them is problematic. It seems to be a duplicate of https://github.com/shutter-project/shutter/issues/271 which is unresolved as well though. Which desktop are you guys using?

**Comment by gapys-krzysztof** [comment]
Hello,

I really love the app, been using it on another machine for quite some time, however cannot get it to work in any form on the current one. The logs seem almost identical, though a bit longer on my side, so I think I should post them too
https://pastebin.com/RXSJc1Us

"Which desktop are you guys using?"
I'm running  Arch Linux x86_64, 5.15.6-arch2-1 using DWM as the window manager.

Any help would be appreciated!

**Comment by Photon89** [comment]
@gapys-krzysztof Did you use DWM on the working machine as well?

**Comment by gapys-krzysztof** [comment]
Sadly not, there it's Manjaro with XFCE, and I believe Shutter is installed by default

**Comment by Photon89** [comment]
I assume that the desktop is the problem here. I'm in a hurry, but we would need to find out how DWM handles .desktop files.

**Comment by gapys-krzysztof** [comment]
It does support and run them, if that is what you're asking.

Though I don't think it's the issue, as I tried both getting the package from the package manager and later building it myself from this repository and running the executable directly (should've mentioned it before, apologies), to the same result.

**Comment by Photon89** [comment]
Does the desktop have the capacity to display tray icons and does this work for other apps?

**Comment by gapys-krzysztof** [comment]
By default DWM doesn't provide any means to display tray icons, though the idea of the window manager is whoever wants it, can patch it into it. https://dwm.suckless.org/patches/systray/

I have the patch installed and enabled. Before I had the patch, I used Discord and Telegram, both provide a systray icon/menu by default and worked with and without the patch (just made it a bit inconvenient to use without a tray icon)

**Comment by Photon89** [comment]
Comparing with the output which I get on a working machine, this piece seems to be something problematic:
```

Can't use an undefined value as an ARRAY reference at /usr/local/bin/shutter line 8346, <DATA> line 19.
        Shutter::App::fct_ret_window_menu() called at /usr/local/bin/shutter line 8449
        Shutter::App::fct_ret_tray_menu() called at /usr/local/bin/shutter line 473
        Shutter::App::STARTUP(Shutter::App=HASH(0x55a004477128)) called at /usr/lib/perl5/5.34/vendor_perl/Glib/Object/Introspection.pm line 67
        Glib::Object::Introspection::__ANON__(Shutter::App=HASH(0x55a004477128)) called at /usr/local/bin/shutter line 10950
```

@DarthGandalf Is this output indicating some problem?

**Comment by gapys-krzysztof** [comment]
```
		#add all windows to menu to capture it directly
		#foreach my $win (@{$wnck_screen->get_windows_stacked}) {
			# if ($active_workspace && $win->is_on_workspace($active_workspace)) {
			# 	my $win_name = $win->get_name;
			# 	Encode::_utf8_on($win_name);
			# 	my $window_item = Gtk3::ImageMenuItem->new_with_label($win_name);
			# 	foreach my $child ($window_item->get_children) {
			# 		if ($child =~ /Gtk3::AccelLabel/) {
			# 			$child->set_width_chars(50);
			# 			$child->set_ellipsize('middle');
			# 			last;
			# 		}
			# 	}
			# 	$window_item->set_image(Gtk3::Image->new_from_pixbuf($win->get_mini_icon));
			# 	$window_item->set('always_show_image' => TRUE);
			# 	$window_item->signal_connect(
			# 		'activate' => \&evt_take_screenshot,
			# 		"shutter_window_direct" . $win->get_xid
			# 	);
			# 	$menu_windows->append($window_item);
			# }
		#}
```

Commenting out this part makes the app launch but I cannot make screenshots of
Desktop ("Error while taking the screenshot. Keyboard could not be grabbed.") or
Windows ("Error while taking the screenshot. Maybe mouse pointer could not be grabbed or the selected area is invalid.") 
with it (probably understandably so, since I removed code).

https://i.imgur.com/6AQFUXL.png Selection screenshots work, though 🎉 

Considering the actual errors turn out to be at the end of the logs, my problem seems different from OP's after all, should I make a new issue ticket?

**Comment by Photon89** [comment]
Yes, sounds like a good idea as the issue is not related to launching via a .desktop file specifically.

Looks like the issue is due to how DWM handles windows and/or workspaces, so maybe it is worth getting in touch with DWM devs as well to get some useful hints.
